src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -181,7 +181,9 @@ private function parseImports(array $content, $file)
         $defaultDirectory = dirname($file);
         foreach ($content['imports'] as $import) {
             if (!is_array($import)) {
-                throw new InvalidArgumentException(sprintf('The values in the "imports" key should be arrays in %s. Check your YAML syntax.', $file));
+                $import = array('resource' => $import);
```

Comment:
Do we really care? Whatever the input is; it crashes like before.

```yml
imports:
  - { resource: { breakit: config.yml } }
```

```
Warning: strlen() expects parameter 1 to be string, array given
```

Which actually comes from `FileLoader::glob()`. AFAIK the  api describes type `mixed` for resources. Hence i think we should _just_ pass here... and fix the glob issue of course ^^